assets/tasks/common.json

```diff
@@ -845,7 +845,14 @@
                 ".store-__container__/",
                 "${config:torizon_login}@${config:torizon_ip}:/home/${config:torizon_login}/.store-__container__"
             ],
-            "dependsOrder": "sequence"
+            "dependsOrder": "sequence",
+            "presentation": {
+                "echo": true,
+                "reveal": "always",
+                "focus": true,
+                "panel": "shared",
```

Comment:
`echo` `reveal` and `panel` are using the default values, please remove them from here

Reply:
Sure